versioning/BRM/extentmap.cpp

```diff
@@ -96,7 +96,7 @@ namespace
 unsigned ExtentSize = 0; // dmc-need to deprecate
 unsigned ExtentRows              = 0;
 unsigned filesPerColumnPartition = 0;
-unsigned extentsPerSegmentFile   = 0;
+unsigned extentsPerSegmentFile   = 2;
```

Comment:
We should use the constant defined in dbcon/joblist/columncommand-jl.h  if this doesn't make a circular dependency.